src/mavsdk/core/CMakeLists.txt

```diff
@@ -121,7 +121,7 @@ if(ANDROID)
 endif()
 
 if((BUILD_STATIC_MAVSDK_SERVER AND ("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")) OR
-    (${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "(armv6|armv7)"))
+    (${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "(armv6|armv7|aarch64)"))
```

Comment:
Interesting, is this now suddenly required for aarch64 as well?

Reply:
This was to make it work on the raspberry pi. Without the statement, it wouldn't include the needed library, if I remember correctly.

Reply:
Ok, that's possible.